packages/node/test/projects/2/package.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "test-2",
   "scripts": {
-    "setup": "npm install && patch-package"
+    "setup": "patch-package"
   },
   "dependencies": {
     "keccak": "3.0.0"
   },
   "devDependencies": {
-    "patch-package": "^6.2.2"
+    "patch-package": "6.2.2"
```

Comment:
I'm going to go with "no" for now, because it doesn't work on my machine:

```
Command failed: /Users/boneskull/projects/lavamoat/lavamoat/packages/node/test/../src/cli.js ./index.js
    Error: No native build was found for platform=darwin arch=arm64 runtime=node abi=115 uv=1 armv=8 libc=glibc node=20.9.0
        loaded from: /Users/boneskull/projects/lavamoat/lavamoat/packages/node/test/projects/2/node_modules/keccak
    
      at load.resolve.load.path (/Users/boneskull/projects/lavamoat/lavamoat/packages/node/test/projects/2/node_modules/node-gyp-build/node-gyp-build.js:61:9)
      at Object.eval (/Users/boneskull/projects/lavamoat/lavamoat/packages/node/test/projects/2/node_modules/keccak/bindings.js:2:73)
      at internalRequire (LavaMoat/core/kernel:896:27)
      at requireRelative (LavaMoat/core/kernel:941:27)
      at requireRelativeWithContext (LavaMoat/core/kernel:909:18)
      at Object.eval (/Users/boneskull/projects/lavamoat/lavamoat/packages/node/test/projects/2/node_modules/keccak/index.js:2:18)
      at internalRequire (LavaMoat/core/kernel:896:27)
      at requireRelative (LavaMoat/core/kernel:941:27)
      at requireRelativeWithContext (LavaMoat/core/kernel:909:18)
      at Object.eval (/Users/boneskull/projects/lavamoat/lavamoat/packages/node/test/projects/2/index.js:11:26)
```